serving/resource_model_serving.go

```diff
@@ -24,33 +21,16 @@ func ResourceModelServing() common.Resource {
 			m["name"].ForceNew = true
 			common.MustSchemaPath(m, "config", "served_models").ConflictsWith = []string{"config.served_entities"}
 			common.MustSchemaPath(m, "config", "served_entities").ConflictsWith = []string{"config.served_models"}
-			common.MustSchemaPath(m, "config", "served_models", "scale_to_zero_enabled").Required = false
-			common.MustSchemaPath(m, "config", "served_models", "scale_to_zero_enabled").Optional = true
-			common.MustSchemaPath(m, "config", "served_models", "scale_to_zero_enabled").Default = true
```

Comment:
Is it intentional to remove this? I believe `scale_to_zero_enabled` is required in `served_models` blocks after this change. See the 2nd and 3rd lines of the diff at the very bottom of "Run make diff-schema" https://github.com/databricks/terraform-provider-databricks/actions/runs/9888048187/job/27311076464?pr=3690:
```
@ ["provider_schemas","registry.terraform.io/databricks/databricks","resource_schemas","databricks_model_serving","block","block_types","config","block","block_types","auto_capture_config","block","attributes","table_name_prefix","computed"]
+ true
@ ["provider_schemas","registry.terraform.io/databricks/databricks","resource_schemas","databricks_model_serving","block","block_types","config","block","block_types","served_models","block","attributes","scale_to_zero_enabled","optional"]
- true
@ ["provider_schemas","registry.terraform.io/databricks/databricks","resource_schemas","databricks_model_serving","block","block_types","config","block","block_types","served_models","block","attributes","scale_to_zero_enabled","required"]
+ true
@ ["provider_schemas","registry.terraform.io/databricks/databricks","resource_schemas","databricks_model_serving","block","block_types","config","block","block_types","served_models","block","attributes","workload_type","computed"]
+ true
```

Reply:
Hmm yeah I see where the discrepancy is, the backend doesn't actually require this parameter anymore, but it seems the OpenAPI spec hasn't been updated to reflect that. Reverting these lines for now, and will update again once the SDK changes land